src/sagemaker/image_uris.py

```diff
@@ -105,6 +121,8 @@ def retrieve(
         py_version,
         container_version,
     )
+
```

Comment:
Hi Jeniya, can you get rid of these additional newlines?

Reply:
updated it